Apps/JobScheduler/src/Jobs/NotificationSettingsJob.cs

```diff
@@ -74,51 +74,47 @@ public class NotificationSettingsJob : INotificationSettingsJob
         public void PushNotificationSettings(string notificationSettingsJson)
         {
             this.logger.LogDebug("Queueing Notification Settings push to PHSA...");
-            if (this.jobEnabled)
+            if (!this.jobEnabled)
```

Comment:
This is quite dangerous - if a job is disabled, it should be disabled from being scheduled by Hangfire

Reply:
I made no change to the logic here, just inverted the if statements to reduce indentation, which reduced cognitive complexity.